models/pattern/patterns/application/argo.go

```diff
@@ -146,10 +146,11 @@ func createNativeArgoResource(opt RolloutEngineGenericOptions) v1alpha1.Rollout
 		}
 
 		containers = append(containers, v1.Container{
-			Name:  container.Name,
-			Image: container.Image,
-			Ports: ports,
-			Env:   container.Envs,
+			Name:    container.Name,
+			Image:   container.Image,
+			Ports:   ports,
+			Env:     container.Envs,
+			Command: container.Commands,
```

Comment:
That addition is probably for adding a new field to rjsf forms for containers inside application? That looks sweet.

Reply:
Yes, that's right @Abhishek-kumar09. Support for executing custom commands was added in this PR as well. I forgot to mention that in the PR description.